client/post-editor/editor-revisions-list/style.scss

```diff
@@ -40,50 +93,59 @@
 }
 
 .editor-revisions-list__date {
-	font-weight: bold;
-}
-
-.editor-revisions-list__changes {
-	padding-top: 5px;
-	font-size: 11px;
+	color: $gray-text;
+	font-weight: 600;
+	display: block;
+	margin-bottom: 4px;
 }
 
-.editor-revisions-list__additions {
-	color: $blue-medium;
+.editor-revisions-list__author {
+	font-size: 13px;
+	display: block;
+	margin-bottom: 4px;
+	color: $gray-text-min;
 }
 
-.editor-revisions-list__deletions {
-	color: $alert-red;
+.editor-revisions-list__changes {
+	font-size: 13px;
+	display: block;
 }
 
+.editor-revisions-list__additions,
+.editor-revisions-list__deletions,
 .editor-revisions-list__minor-changes {
-	color: $gray-text-min;
-}
-
-.editor-revisions-list__scroller {
-	padding: 10px 0;
-	height: 100%;
-	overflow-y: auto;
-	overflow-x: hidden;
-	-webkit-overflow-scrolling: touch;
-}
+	line-height: 1;
 
-.editor-revisions-list__list,
-.editor-revisions-list__header {
-	background: $white;
-	border-top: 1px solid darken($sidebar-bg-color, 5%);
-	border-bottom: 1px solid darken($sidebar-bg-color, 5%);
-	font-size: 13px;
+	b {
+		display: inline-block;
+		border-radius: 12px;
+		padding: 0;
+		line-height: 12px;
+		font-size: 14px;
+		height: 14px;
+		width: 14px;
+		color: white;
+		text-align: center;
+		margin: 0 6px 0 0;
+		font-weight: normal;
+		box-sizing: border-box;
+	}
 }
 
-.editor-revisions-list__list {
-	border-top: none;
+.editor-revisions-list__additions {
+	margin-right: 12px;
+	b {
+		background: $blue-medium;
+	}
 }
 
-.editor-revisions-list__header {
-	padding: 1rem;
+.editor-revisions-list__deletions {
+	b {
+		background: $alert-red;
+	}
 }
 
-.editor-revisions-list__load-revision {
-	width: 100%;
-}
+.editor-revisions-list__minor-changes {
+	top: 8px;
+	color: $gray-text-min;
+}
```

Comment:
Minor thing but can we all update our editors to make a newline at the end of files? It helps diffs and other code manipulation. We make use of http://editorconfig.org in Calypso so it should be very easy :)